src/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .app_page_scraper import PlayAppPageScraper
 from .betel_errors import BetelError, PlayScrapingError, AccessError
+from .utils import SCRAPER_INFO_FILE_NAME, SCRAPER_LOG_FILE_NAME, get_app_icon_name
```

Comment:
global comment:

The style guide says:  
> Use import statements for packages and modules only, not for individual classes or functions. Note that there is an explicit exemption for imports from the typing module.

so if you want to follow it, most of the imports in this project would need to be restructured. But I think it would also be OK to leave as is, perhaps consult Seb or Momo for a decision.

Reply:
As per our discussion, perhaps this is fine for the public re-export of some things in the __init__.py (even though the style guide doesn't explicitly allow it), but the comment still applies to the other files.